azure-servicebus/pom.xml

```diff
@@ -3,7 +3,7 @@
 	<description>Java library for Azure Service Bus</description>	
 	<modelVersion>4.0.0</modelVersion>		
 	<artifactId>azure-servicebus</artifactId>	
-	<version>2.0.0-PREVIEW-7</version>
+	<version>${client-current-version}</version>
```

Comment:
>>${client-current-version}< [](start = 9, length = 27)

This had to be removed as this was not allowed with the latest release infra I believe.